java/release_notes.txt

```diff
@@ -21,9 +21,12 @@ Metadata changes:
    269 (fr), 238(pt), 55 (pt)
  - Updated carrier data for country calling code(s):
    91 (en), 225 (en), 234 (en), 235 (en), 675 (en), 993 (en)
+Other changes:
  - First release to automatically remove obsolete geocoding files
 
 Jan 08, 2016: libphonenumber-7.2.3
+Backwards-incompatible code changes:
```

Comment:
I would put code changes and "backwards incompatible code changes" beside each other. (Also: next time we should make this v8.0 if there are such changes.)